### ISSUE TITLE
how to get cookies ?

### ISSUE THREAD

**Issue by widnyana** [ysmood/rod#18]
hello, 

I'm struggling on getting cookies from the page. I've seen on how `rod` sets the cookie using `*rod.Page` and see this [doc](https://chromedevtools.github.io/devtools-protocol/tot/Network/#method-getCookies); but I cant manage to pull all the cookie from given url.

Here's the snippet:

```go
func RodGetCookies(p *rod.Page, urls []string) {
	result := p.Call("Network.getCookies", cdp.Object{
		"urls": urls,
	})

	fmt.Printf("%+v", result)
}

...
    page, err := browser.PageE("https://github.com")
	if err != nil {
		panic(err)
	}

	RodGetCookies(page, []string{"*.github.com"})
...
```

the output:

```
{"cookies":[]}
```

thank you

**Comment by widnyana** [comment]
@ysmood thank you for the suggestion :)

I already try that actually, and got error like this:

```
panic: [rod] DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.
    at <anonymous>:1:32
{"result":{"type":"object","subtype":"error","className":"DOMException","description":"DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.\n    at <anonymous>:1:32","objectId":"{\"injectedScriptId\":1,\"id\":3}","preview":{"type":"object","subtype":"error","description":"DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.\n    at <anonymous>:1:32","overflow":false,"properties":[{"name":"stack","type":"accessor"},{"name":"code","type":"number","value":"18"},{"name":"name","type":"string","value":"SecurityError"},{"name":"message","type":"string","value":"Failed to read the 'cookie' property from 'Document': Access is denied for this document."}]}},"exceptionDetails":{"exceptionId":1,"text":"Uncaught","lineNumber":0,"columnNumber":31,"scriptId":"22","exception":{"type":"object","subtype":"error","className":"DOMException","description":"DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.\n    at <anonymous>:1:32","objectId":"{\"injectedScriptId\":1,\"id\":4}","preview":{"type":"object","subtype":"error","description":"DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.\n    at <anonymous>:1:32","overflow":false,"properties":[{"name":"stack","type":"accessor"},{"name":"code","type":"number","value":"18"},{"name":"name","type":"string","value":"SecurityError"},{"name":"message","type":"string","value":"Failed to read the 'cookie' property from 'Document': Access is denied for this document."}]}}}}

goroutine 1 [running]:
github.com/ysmood/kit/pkg/utils.E(0xc0001d2af0, 0x1, 0x1, 0x0, 0xe24b75, 0xf)
        /home/widnyana/.gvm/pkgsets/go1.13.5/global/pkg/mod/github.com/ysmood/kit@v0.22.2/pkg/utils/utils.go:33 +0xab
github.com/ysmood/rod.(*Page).Eval(0xc000172000, 0xe24b75, 0xf, 0x0, 0x0, 0x0, 0x1)
        /home/widnyana/.gvm/pkgsets/go1.13.5/global/pkg/mod/github.com/ysmood/rod@v0.25.7/sugar.go:232 +0xf8

```

---

long story short, I can make my snippet above works by adding code below before calling `RodGetCookies`

```go
el := page.WaitLoad().Element("#topsection > div > h1")
elStable := meh.WaitStable().WaitVisible()  // <-- this one
fmt.Println(elStable.Text())
```

closing this issue, and again thank you for creating this library, cheers! :beers:

**Comment by ysmood** [comment]
Released `v0.27.3`, now you have `Cookies` to get the cookies directly:

https://github.com/ysmood/rod/blob/52fc1e08901b8ad7a11856c6bb888c67bc870b76/page_test.go#L33

**Comment by widnyana** [comment]
thank you @ysmood !
